bots/pr/src/main/java/org/openjdk/skara/bots/pr/ReviewersCommand.java

```diff
@@ -145,7 +145,7 @@ public void handle(PullRequestBot bot, PullRequest pr, CensusInstance censusInst
         // Create a helpful message regarding the required distribution (if applicable)
         var nonZeroDescriptions = formatLimits.entrySet().stream()
                 .filter(entry -> entry.getValue() > 0)
-                .map(entry -> entry.getValue() + " " + entry.getKey() + (entry.getValue() > 1 ? "s" : ""))
+                .map(entry -> entry.getValue() + " " + String.format(entry.getKey(), entry.getValue() > 1 ? "s" : "", "http://openjdk.java.net/bylaws"))
```

Comment:
This really should use "https" rather than "http".

Reply:
It is good to fix it at this PR. I am fixing it now.

Reply:
Done.